quickstarts/kit/custom-build-plugin/README.md

```diff
@@ -0,0 +1,10 @@
+---
+name: "Kit :: Maven :: Custom Build Plugin Parent"
+description: |
+A simple project demonstrating how to write your own custom build plugins using Eclipse JKube.
+One module is build plugin and other is app module consuming the custom build plugin.
+---
+
+This project contains two submodules:
+- [App](./app)
+- [Foo Build Plugin](./foo-build-plugin)
```

Comment:
I think it might be reasonable to merge both READMEs in this one with appropriate headings (H2).

It will be easier to follow for users.

Reply:
I'd also repurpose the build plugin and the consuming app.

We could use the build plugin to retrieve data from an online service and populate a file.
This file, or its contents, could then be served by the final application. So each time the application was built and deployed, the file would be regenerated. This is exactly what we're doing (using JavaScript) in the JKube website when we process the asciidoc documentation, or the quickstarts.

In general, this would resemble more what a real use-case would look like.

For example, a real use case that would be useful for us (please don't implement this) in JKube would be script that packaged a single quickstart in a zip file so that it could be then downloaded from the main application web service.